webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -100,13 +105,16 @@ export default {
       },
       formSchema: {
         title: { required: true, min: 3, max: 64 },
-        content: { required: true, min: 3 },
+        content: [{ required: true }],
       },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T09:09:56Z" title="Thursday, July 18th 2019, 11:09:56 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -177,6 +181,14 @@ export default {
     updateEditorContent(value) {
       // this.form.content = value
       this.$refs.contributionForm.update('content', value)
+
+      this.disabled = true
+      this.n = value.replace(/<\/?[^>]+(>|$)/gm, '').length
+      this.form.contentLength = this.n
+
+      if (this.n > this.formSchema.content.min && this.n < this.formSchema.content.max) {
+        this.disabled = false
```


`this.n` should be a local `let` variable and not be global to this component …

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-18T19:14:21Z" title="Thursday, July 18th 2019, 9:14:21 pm +02:00">Jul 18, 2019</time>_

---


 amended